src/textEditor.ts

```diff
@@ -140,6 +141,32 @@ export class TextEditor {
     return vscode.window.activeTextEditor.document.getText(selection);
   }
 
+  /**
+   *  Retrieves the current word at position.
+   *  If current position is whitespace, selects the right-closest word
+   */
+  static getWord(position: Position) : string {
+    const whitespaceRegex = new RegExp("^ *$");
```

Comment:
I was kinda feeling like maybe you would want to, but I can't think of any concrete cases, so yeah, let's not worry about that for now. 

Can you just dump it on here then? https://github.com/VSCodeVim/Vim/blob/master/src/globals.ts